emails/templates/passwordless_verify_existing_user.txt

```diff
@@ -0,0 +1,6 @@
+[[HiddenSubject .Subject "Verify your email"]]
+
+Hi,
+
+Copy and paste the following link directly in your browser to verify your email. This link will expire in {{ .Expire }} minutes.
+[[.AppUrl]]login/?code=[[.Code]]&confirmationCode=[[.ConfirmationCode]]
```

Comment:
Do you want to have this mirror the html email and lead with the ability to enter the code?

Reply:
Yeah, I just pushed a commit to improve the copy in these `txt` email templates.